client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -316,7 +317,8 @@ static Request bulk(BulkRequest bulkRequest) throws IOException {
                 BytesReference indexSource = indexRequest.source();
                 XContentType indexXContentType = indexRequest.getContentType();
 
-                try (XContentParser parser = XContentHelper.createParser(NamedXContentRegistry.EMPTY, indexSource, indexXContentType)) {
+                try (XContentParser parser = XContentHelper.createParser(NamedXContentRegistry.EMPTY,
+                    LoggingDeprecationHandler.INSTANCE, indexSource, indexXContentType)) {
```

Comment:
Can you indent this one more time? The way it is the method args look like they are part of the `try` body.

Reply:
I checked but I thought we used what Intellij thought the indentation should be? (I even opened Intellij to make sure it agreed)

Reply:
I tend to disagree with IntelliJ here. It is hard to read..... Whatever works though.